src/sentry/api/endpoints/organization_sampling_project_span_counts.py

```diff
@@ -0,0 +1,75 @@
+from collections.abc import Sequence
+from typing import cast
+
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import OrganizationEndpoint, OrganizationPermission
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.utils import get_date_range_from_params
+from sentry.constants import ObjectStatus
+from sentry.models.organization import Organization
+from sentry.models.project import Project
+from sentry.sentry_metrics.querying.data import (
+    MetricsAPIQueryResultsTransformer,
+    MQLQuery,
+    run_queries,
+)
+from sentry.sentry_metrics.querying.types import QueryOrder, QueryType
+from sentry.snuba.metrics import SpanMRI
+from sentry.snuba.referrer import Referrer
+from sentry.utils.dates import parse_stats_period
+
+
+@region_silo_endpoint
+class OrganizationSamplingProjectSpanCountsEndpoint(OrganizationEndpoint):
+    """Endpoint for retrieving project span counts in all orgs."""
+
+    owner = ApiOwner.TELEMETRY_EXPERIENCE
+    permission_classes = (OrganizationPermission,)
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
+    }
+
+    def get(self, request: Request, organization: Organization) -> Response:
+        self._check_feature(request, organization)
+
+        start, end = get_date_range_from_params(request.GET)
+        projects = cast(
+            Sequence[Project],
+            list(
+                Project.objects.filter(organization=organization, status=ObjectStatus.ACTIVE).all()
+            ),
+        )
```

Comment:
Cast to `Sequence` should no longer be required - a list is a sequence. Also, `list()` will evaluate the queryset, I think you can omit the `.all()`.

Optionally, I'd add a comment to clarify that we do not want team filters even with closed membership, so that in the future this is not unintentionally "fixed".

```suggestion
        projects = list(Project.objects.filter(organization=organization, status=ObjectStatus.ACTIVE))
```